src/Search.elm

```diff
@@ -326,8 +354,8 @@ update toRoute navKey msg model =
 
         QueryResponse result ->
             -- let
-                -- _ =
-                --     Debug.log "got query result" result
+            -- _ =
+            --     Debug.log "got query result" result
             -- in
```

Comment:
```suggestion
```

Reply:
I am aware of these and will remove all the debug lines soon before this gets merged

Reply:
yeah sorry. Frankly I mistaken the repo for something else.